drake/util/Polynomial.h

```diff
@@ -10,9 +10,30 @@
 #include <stdexcept>
 #include "drake/drakePolynomial_export.h"
 
-// represents a scalar multi-variate polynomial
-// modeled after the msspoly in spotless
-
+/// A scalar multi-variate polynomial, modeled after the msspoly in spotless.
```

Comment:
This assumes autobrief is enabled in Drake's doxygen -- if not it should be but I've seen lots of `@brief`s in the code; did the briefs come out as you wanted them?